### ISSUE TITLE
64bit Windows - RuntimeError: buffer has unexpected item byte size (8, expected 4)

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#4751]
Recently the AppVeyor tests have started to fail, first noticed on PR #4745 but other PRs since like #4749 and #4750. This appears to be a platform change, rather than a code change on our part.

e.g. https://ci.appveyor.com/project/biopython/biopython/build/job/v3r88ery4an0kec5 

Multiple failures like this:

```
----------------------------------------------------------------------
File "C:\Py\Lib\site-packages\Bio\Phylo\TreeConstruction.py", line 1229, in Bio.Phylo.TreeConstruction.ParsimonyTreeConstructor
Failed example:
    alignment = Align.read(open('TreeConstruction/msa.phy'), 'phylip')
Exception raised:
    Traceback (most recent call last):
      File "C:\Py\Lib\doctest.py", line 1361, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest Bio.Phylo.TreeConstruction.ParsimonyTreeConstructor[11]>", line 1, in <module>
        alignment = Align.read(open('TreeConstruction/msa.phy'), 'phylip')
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Py\Lib\site-packages\Bio\Align\__init__.py", line 4367, in read
        alignment = next(alignments)
                    ^^^^^^^^^^^^^^^^
      File "C:\Py\Lib\site-packages\Bio\Align\interfaces.py", line 75, in __next__
        alignment = self._read_next_alignment(stream)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Py\Lib\site-packages\Bio\Align\phylip.py", line 179, in _read_next_alignment
        seqs, coordinates = Alignment.parse_printed_alignment(seqs)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "C:\Py\Lib\site-packages\Bio\Align\__init__.py", line 1106, in parse_printed_alignment
        parser.fill(coordinates)
    RuntimeError: buffer has unexpected item byte size (8, expected 4)
----------------------------------------------------------------------
File "C:\Py\Lib\site-packages\Bio\Phylo\TreeConstruction.py", line 1230, in Bio.Phylo.TreeConstruction.ParsimonyTreeConstructor
```

**Comment by peterjc** [comment]
Not just AppVeyor, also GitHub Actions

**Comment by mdehoon** [comment]
Fixed by #4752
